### ISSUE TITLE
Where did you download the vgg16 pretrained weight?

### ISSUE THREAD

**Issue by lauraset** [YonghaoXu/CRGNet#1]
Hi, @YonghaoXu. Your work is great and inspires me lot. 
I want to know the link that you download the vgg16 pretrained weight.
I notice that pytorch has already provided a vgg16 pretrained weight in `models.vgg16(pretrained=True)`.
So I am curious about why you use another pretrained weight.
Thank you very much.

**Comment by YonghaoXu** [comment]
Hi @lauraset, thanks for your interest in this work. The pretrained VGG-16 model is the same one that I used in a previous project on cross-domain semantic segmentation. You can download it [here](https://drive.google.com/file/d/1PGuOb-ZIOc10aMGOxj5xFSubi8mkVXaq/view?usp=sharing). I think directly using the official pytorch pretrained weight would also be fine as here we just want to have a good initialization to accelerate the training procedure.

**Comment by lauraset** [comment]
Hi, @YonghaoXu. Thank you for your detailed reply. I got it.

**Comment by lauraset** [comment]
Hi, @YonghaoXu. Thank you for your detailed reply. I got it.
By the way, I used the well-trained weights of CRGNet. 
When I run `GenVaihingen.py`, the results show that
![image](https://user-images.githubusercontent.com/39206462/206695496-a0d59899-e772-423e-bfa2-95c9014af131.png)
The boundary seems poor. Is that right?

**Comment by YonghaoXu** [comment]
Yes, since CRGNet only uses point-level annotations, the generated pseudo labels may not be accurate, especially on the boundary regions. Thus, we propose to use the self-training technique to finetune the model with these inaccurate pseudo labels. The dense CRF can also help to further improve the accuracy of those boundary regions.

**Comment by lauraset** [comment]
Hi, @YonghaoXu. Thank you very much. I have obtained the similar results by self-training as reported in your paper. It is a practical technique.
